src/app/static/src/js/components/runWorkflow/runWorkflowTabs.vue

```diff
@@ -49,13 +49,21 @@ export default Vue.extend({
     name: "runWorkflowTabs",
     data() {
         return {
-            selectedTab: "runWorkflow"
+            selectedTab: "runWorkflow",
+            selectedWorkflow: ""
         }
     },
     methods: {
         switchTab(tab) {
             this.selectedTab = tab
-        }
+        },
+        viewProgress(workflowKey) {
+            this.switchTab('runWorkflowProgress');
+            this.selectedWorkflow = workflowKey;
```

Comment:
I think logically you want to set the selected workflow before switching tab to mount the progress component, though of course in practice it doesn't seem to make a difference!

Reply:
Yeah, I noticed that afterwards, but it was working so I didn't change it. Have switched it now though.